app/routes/$repo/$.tsx

```diff
@@ -156,9 +156,9 @@ export default function RepoDocument() {
   //
   // Logically a tool is NOT included in the list of flow routes (routes with the parent /flow)
   // const tool = ![...ROUTING_STRUCTURE.flow].includes(content.contentName)
-  const isSwitchContent = Object.keys(switchContents).includes(
-    content.contentName
-  )
+  const isSwitchContent =
+    Object.keys(switchContents).includes(content.contentName) ||
+    content.contentName === "flow"
```

Comment:
Chose to put it here because if I added a "flow" key to `switchContents` it wouldn't make sense since we don't want an item to show up in the menu for `/flow`